extensions/sns/extension.json

```diff
@@ -77,6 +77,17 @@
         }
       }
     },
+    "neuron-id-to-candid-subaccount": {
+      "about": "Converts a Neuron ID to a candid subaccount blob suitable for use in the `manage_neuron` method on SNS Governance",
+      "args": {
+        "neuron_id": {
+          "about": "The (hex-encoded) SNS neuron ID to convert to a candid subaccount blob"
+        },
```

Comment:
This is a positional argument, I wasn't sure how to denote that

Reply:
Not having `long` or `short` should indicate that this is positional (so no change needed)